stickynotes/stickynotes_applet_callbacks.c

```diff
@@ -27,59 +27,58 @@
 
 static gboolean get_desktop_window (Window *window)
 {
-	Window *desktop_window;
-	GdkWindow *root_window;
-	GdkAtom type_returned;
-	int format_returned;
-	int length_returned;
-
-	root_window = gdk_screen_get_root_window (
-				gdk_screen_get_default ());
-
-	if (gdk_property_get (root_window,
-			      gdk_atom_intern ("CAJA_DESKTOP_WINDOW_ID", FALSE),
-			      gdk_x11_xatom_to_atom (XA_WINDOW),
-			      0, 4, FALSE,
-			      &type_returned,
-			      &format_returned,
-			      &length_returned,
-			      (guchar**) &desktop_window)) {
-		*window = *desktop_window;
-		g_free (desktop_window);
-		return TRUE;
-	}
-	else {
-		*window = 0;
-		return FALSE;
-	}
+    Window *desktop_window;
+    GdkWindow *root_window;
+    GdkAtom type_returned;
+    int format_returned;
+    int length_returned;
+
+    root_window = gdk_screen_get_root_window (
+        gdk_screen_get_default ());
```

Comment:
there is no need to break the line because the it's not very long, however if you think so, it's better do it at the equal sign, leaving a parenthesis open alone is a bad idea

Reply:
You're right, i forgot to change that, it was there before.
Author of this code did break lines like this very often.